study-builder/studies/angio/study.conf

```diff
@@ -31,6 +31,8 @@
 
     "studyDetails": [],
 
+    "studySupportedLanguages": [ "en" ],
```

Comment:
Can we call it `supportedLanguages`? Having `study` prefix seems redundant since we already know it's a config file for a study.

Reply:
Done.